app/scripts/convert.py

```diff
@@ -47,7 +48,23 @@ def to_keys_indexes_str(_input):
 
 def convert(file_path):
```

Comment:
You can tell flake8 to ignore this line with inline comment `# noqa: C901`
The `C901` is from the pre-commit check near the bottom of the PR.
[Reference](https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html)